goal_src/levels/swamp/swamp-rat.gc

```diff
@@ -5,3 +5,920 @@
 ;; name in dgo: swamp-rat
 ;; dgos: L1, SWA
 
+(define-extern *swamp-rat-sg* skeleton-group)
+(define-extern pointer type) ;; ...why is this needed
```

Comment:
Yeah, it can actually be removed from the .gc, but if its not in `all-types` one of the functions does not decompile properly:
```
;; WARN: Type Propagation failed: Failed type prop at op 21 ((set! a0 pointer)): Do not have the type of symbol pointer
;; WARN: Type Propagation failed: Type analysis failed
(defmethod
  dummy-44
  swamp-rat
```

Reply:
ah, I forgot about that. Near the top of `all-types.gc` there's a bunch of stuff like `(define-extern int16 type)`, we should just put it there.  The type system can't easily define symbols, so we have to do it manually.